core/block/import/objectidgetter.go

```diff
@@ -111,14 +110,14 @@ func (ou *ObjectIDGetter) getObjectByOldAnytypeID(sn *converter.Snapshot, sbType
 	return "", err
 }
 
-func (ou *ObjectIDGetter) getSubObjectID(sn *converter.Snapshot, oldToNewIDs map[string]string) (string, error) {
-	ids, err := ou.getAlreadyExistingSubObject(sn, oldToNewIDs)
```

Comment:
Function return nil as error in all cases. So I remove error from returned param, because it's always nil